src/com/itacademy/eshop/simulations/CustomerSimulation.java

```diff
@@ -29,21 +31,32 @@ private void addProductsToShoppingCart() {
          * User browses the shop and adds 4 products to the shopping cart.
          * One of them should be a book and other should be a shirt.
          */
-        throw new UnsupportedOperationException("addProductsToShoppingCart() is not implemented yet");
+//        Product foundProduct = shop.findProductByName("shirt"); just note to myself
```

Comment:
Not to get into a habit of having commented code. Remove it unless it is a TODO flag